### PR TITLE
Update LogarithmicScaleDown featuregate docs

### PR DESCRIPTION
**Pull request by damemi**
Ref https://github.com/kubernetes/enhancements/issues/2185
This updates the `LogarithmicScaleDown` feature gate documentation to reflect its 1.22 Beta status

PR to make this change: https://github.com/kubernetes/kubernetes/pull/101767

### CODE REVIEW

**Comment by tengqm** [comment]
Please add a reference to the upstream change PR so that we know if and when this one can be kicked in. Thanks.
Feel free to lift the hold when this is ready.
/hold

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (soltysh) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:139]: Don't change it, but add a 2nd row for Beta version like above.

**Review by soltysh — CHANGES_REQUESTED** [review]

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (damemi) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:140]: Updated

**Review by soltysh — APPROVED** [review]
/lgtm

**Comment by PI-Victor** [comment]
/hold 
till this enhancement is not labelled 'At Risk' anymore

**Comment by damemi** [comment]
@PI-Victor this isn't at risk anymore, the PR that was holding it (in https://github.com/kubernetes/enhancements/issues/2185#issuecomment-874951777) merged

**Comment by damemi** [comment]
/hold cancel
the changes for this have merged, ready for approval on the docs update

**Comment by PI-Victor** [comment]
we have an internal spreadsheet that we use to track enhancements, that spreadsheet says this enhancement is marked as at risk, and still does. but no worries, whenever that changes, i will personally merge this, please leave the hold on. if it's only a matter at updating the the spreadsheet, that's also ok. as long as you are set for the code freeze, everything's good.

/hold

**Comment by damemi** [comment]
Ok, sorry @PI-Victor!

**Comment by PI-Victor** [comment]
as promised

/hold cancel
/approve

**Pull request merged**
